gather_cnv_v2v

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+
+for ns in $(oc get namespace $ns --no-headers -o=custom-columns=NAME:.metadata.name 2> /dev/null)
+do
+    echo "NS: $ns"
+    ns_collection_path=$BASE_COLLECTION_PATH/namespaces/$ns/
+    mkdir -p $ns_collection_path
+
+    for pod in $(oc get pods -n $ns --no-headers -o=custom-columns=NAME:.metadata.name 2>/dev/null)
+    do
+        if ! expr $pod : 'kubevirt-v2v-conversion-*' >/dev/null
+        then
+            continue
+        fi
+
+        echo "POD: $pod"
+        pod_collection_path=$ns_collection_path/$pod
+        mkdir -p $pod_collection_path
+
+        /usr/bin/openshift-must-gather inspect -n $ns pod/$pod
+        oc logs $pod -n $ns > $ns_collection_path/$pod.log
```

Comment:
Should this not be $pod_collection_path?

Reply:
@nyoxi Can you confirm if it should effectively be $pod_collection_path here?

Reply:
No, it is `$ns_collection_path` to be consistent with `gather_cnv_network`. I will drop `$pod_collection_path` from the patch as that is unused relic from development.

Reply:
So when I was testing this for the cdi must gather PR, I noticed it would create the directories, but then not put the files in the directory it created. So I ended up with an empty directory. That is why I used $pod_collection_path to put my files in (so the generated files end up in the corresponding directory).

Reply:
I don't have any strong preference here, so whatever suits you best.